src/SearchResults.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { numberOfNight } from "./Nights";
+
+const SearchResults = ({ results }) => {
+  const [selectedRow, setSelectedRow] = useState(null);
+
+  const handleRowClick = (id) => {
+    if (id === selectedRow) {
+      setSelectedRow(null);
+    } else {
+      setSelectedRow(id);
+    }
+  };
```

Comment:
I dont think this will work for this part will it?
```
Test: Verify that each row of your table can be highlighted (on and off) independently when being clicked.
```
How might we make the rows be highlighted independently ?

Reply:
Sorry @sampennington,
I did not understand, at the moment it work. But if it's necessary or not! is that the question?